tests/test_dataset_generator.py

```diff
@@ -91,8 +95,256 @@ def test_generate_custom_dataset(self):
         expected_df = pd.DataFrame.from_dict(
             dict(zip(["int", "dat", "txt", "cat", "flo"], expected_data))
         )
-        df = generate_dataset(
+        df = dg.generate_dataset(
             self.rng,
             columns_to_generate=self.columns_to_gen,
             dataset_length=self.dataset_length,
         )
+        self.assertTrue(df.equals(expected_df))
+
+
+class TestGetOrderedColumn(unittest.TestCase):
+    def setUp(self):
+        self.rng = Generator(PCG64(12345))
+        self.start_date = pd.Timestamp(2001, 12, 22)
+        self.end_date = pd.Timestamp(2023, 1, 1)
+        self.date_format_list = ["%B %d %Y %H:%M:%S"]
+
+    def test_get_ordered_column_datetime_ascending(self):
+        data = dategen.random_datetimes(
+            rng=self.rng, start_date=self.start_date, end_date=self.end_date, num_rows=5
+        )
+
+        ordered_data = np.array(
+            [
+                [
+                    "October 02 2006 22:34:32",
+                    datetime.strptime(
+                        "October 02 2006 22:34:32", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "August 19 2008 16:53:49",
+                    datetime.strptime(
+                        "August 19 2008 16:53:49", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "March 13 2010 17:18:44",
+                    datetime.strptime(
+                        "March 13 2010 17:18:44", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "March 11 2016 15:15:39",
+                    datetime.strptime(
+                        "March 11 2016 15:15:39", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "September 27 2018 18:24:03",
+                    datetime.strptime(
+                        "September 27 2018 18:24:03", self.date_format_list[0]
+                    ),
+                ],
+            ]
+        )
+
+        ordered_data = ordered_data[:, 0]
```

Comment:
probably don't need this.... can just call `ordered_data[:,0]` in `assert_array_equal`